src/resources/views/backup.blade.php

```diff
@@ -79,7 +84,7 @@
                 else
                 {
                     new Noty({
-                        text: "<strong>{{ trans('backpack::backup.create_confirmation_title') }}</strong><br>{{ trans('backpack::backup.create_confirmation_message') }}",
+                        text: "<strong>{{ trans('backpack::backup.create_completed_title') }}</strong><br>{{ trans('backpack::backup.create_confirmation_message') }}",
```

Comment:
Would it be possible to fill the table with the new entry? With an ajax call? Too much work?

Reply:
That would be cool, the feedback here is crap. You don't know if the backing up is in progress... if it's done... if it's failed... it's a bad DX. 

But then again... might be a bit too much work to make this perfect, like you said... for something that only developers will probably use...